designs/EmbeddedReadmesTechSpec.md

```diff
@@ -0,0 +1,102 @@
+# Embedded Readmes Technical Spec
+
+* Status: Implementing
+* Author(s): [Advay Tandon](https://github.com/advay26)
+* Issue: [6873](https://github.com/NuGet/Home/issues/6873) README.md (markdown description) support for NuGet packages
+
+## Problem Background
+
+Users of NuGet want to be able to include Readme files as part of the package when they `pack` and `push` their project through the commandline. The [design document](https://github.com/NuGet/Home/wiki/Packaging-READMEs-within-the-nupkg) outlines the work needed for this feature and the intended results.
+
+## Who are the customers
+
+NuGet customers that use the commandline to publish their packages, and want to be able to include Readme files through the `pack` process, rather than having to do it manually after the package has been published.
+
+## Goals
+
+* Implement the Client side functionality for Embedded Readmes:
+    * nuget pack support
+    * dotnet pack support
+    * Validation of readme files
+
+## Non-Goals
+
+* Server side functionality
```

Comment:
Include as a non-goal: parse readme as a markdown, because:
- we don't have built-in libraries to parse markdown, and adding one would take some time
- There's multiple Markdown implementations (github, daring fireball etc.)